app/controllers/searchs_controller.rb

```diff
@@ -1,16 +1,11 @@
 class SearchsController < ApplicationController
   def new
-    @forums = load_records SubForum
-    @posts = load_records Post
+    @forums = SubForum.search(params[:q])
+    @posts = Post.search(params[:q])
```

Comment:
```suggestion
    @posts = Post.search params[:q]
```
thêm phân trang nếu có nhé

Reply:
trong giao diện mô tả thì bọn em chỉ trích xuất ra 1 số lượng cố định ở trang homepage thôi, nếu người dùng muốn xem thêm thì sẽ sang tab khác, tab đó sẽ có phân trang hoặc sử dụng ajax để load dần ạ